libs/sdk-ui-kit/api/sdk-ui-kit.api.md

```diff
@@ -521,6 +521,9 @@ showOnlyLogoutItem?: boolean, featureFlags?: ISettings): IHeaderMenuItem[];
 // @internal (undocumented)
 export function generateHeaderHelpMenuItems(documentationUrl?: string, supportForumUrl?: string, userEmail?: string, workspaceId?: string, sessionId?: string, supportEmail?: string, isBranded?: boolean, featureFlags?: ISettings): IHeaderMenuItem[];
 
+// @internal
+export function generateHeaderHelpSectionSplitItem(className?: string): IHeaderMenuItem;
```

Comment:
Instead of generating a divider to separate the items, why don't we just add a `divider` class for the item which will separate the groups? So if the item has a `divider` CSS class, it would have a divider line above or below the item.

Reply:
I tried adding a divider class, but it is not easy to make like mockup design
![Screen Shot 2022-07-18 at 13 58 18](https://user-images.githubusercontent.com/5329091/179459251-9bf492f1-1e3d-4337-9eff-64616b3948b5.png)
![Screen Shot 2022-07-18 at 13 58 13](https://user-images.githubusercontent.com/5329091/179459241-15c4e68c-b3c3-4056-800e-c0246e8fef9d.png)

cc: @vannguyengd  @anvuong119

Reply:
fixed